app/src/main/java/org/wikipedia/feed/view/CardLargeHeaderView.java

```diff
@@ -124,4 +126,8 @@ private void setGradientDrawableBackground(@ColorInt int leftColor, @ColorInt in
         gradientDrawable.setCornerRadius(getResources().getDimension(R.dimen.wiki_card_radius));
         borderContainer.setBackground(gradientDrawable);
     }
+
+    public Pair<View, String>[] getSharedElements() {
+        return TransitionUtil.getSharedElements(getContext(), titleView, subtitleView, imageView);
```

Comment:
The transition for this looks pretty good, except for RTL articles. Is there any way to fix it?

Reply:
Good catch! Done.